packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -301,12 +302,72 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
+    regularLabel: [
+      classNames.regularLabel,
+      {
+        position: 'absolute',
+        height: '0px',
+        background: theme.palette.neutralTertiaryAlt,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText'
```

Comment:
All these High Contrast Selectors are styling the `backgroundColor` so perhaps instead of repeating them everywhere you can extract it in a helper function that takes as input the color string and then returns a ready object.

Reply:
where should I write that function in slider.base? or slider.styles?

Reply:
styles file. it can be a global function to that file.

Reply:
It would roughly look like this:
```ts
function getHighContrastSelector(systemColor: string): IRawStyle {
  return {
    selectors: {
      [HighContrastSelector]: {
        backgroundColor: systemColor
      }
   }
}
```

and later just call it with: `getHighContrastSelector('WindowText' or whatever other color you want)`